nvflare/private/fed/server/server_engine.py

```diff
@@ -581,12 +581,16 @@ def aux_send(self, targets: [], topic: str, request: Shareable, timeout: float,
             client_name = self.get_client_name_from_token(r.client_token)
             if r.reply:
                 try:
-                    results[client_name] = fobs.loads(r.reply.body)
-                except BaseException:
+                    shareable = fobs.loads(r.reply.body)
```

Comment:
We should do exactly the same thing as line 733: check return_code first. Only if the RC is OK, should we do fobs.loads().

Reply:
Done.